Detectors/TOF/simulation/include/TOFSimulation/Detector.h

```diff
@@ -72,10 +72,10 @@ class Detector : public o2::Base::DetImpl<Detector>
 
   void CreateMaterials();
   void ConstructGeometry() final;
+  void addAlignableVolumes() const;
```

Comment:
@njacazio Could you please declare this method as
``void addAlignableVolumes() const override;``
and add in this same PR a public method
``virtual void addAlignableVolumes() const {}`` 
to the ``Detectors/Base/include/DetectorsBase/Detector.h``

This method should be eventually called centrally for all detectors in loop, so better to have it virtual.

Reply:
I think it is not the purpose of this PR to define alignable volume interfaces. This PR is about digitization. Maybe we should take out this one commit and do it separately.

Reply:
@sawenzel fine for me, but then this will require one incremental patch to TOF to override the virtual method

Reply:
We can also make a sep PR to introduce the interface change and then rebase this PR on top. We do have this time.

Reply:
Dear @shahor02, together with @noferini we suggest to make the method virtual in another PR after the method is added to the DetImpl class. We can still remove that commit from this PR but it is essentially one bug fix (with only one line removed) which was introduced and I agree that the title of the commit is a bit misleading, @sawenzel  is that OK?